db/migrate/20160129082610_add_object_changes_to_versions.rb

```diff
@@ -0,0 +1,10 @@
+class AddObjectChangesToVersions < ActiveRecord::Migration
+
+  # The largest text column available in all supported RDBMS.
+  # See `create_versions.rb` for details.
+  TEXT_BYTES = 1_073_741_823
+
+  def change
+    add_column :versions, :object_changes, :text, limit: TEXT_BYTES
```

Comment:
Could this be merged into the other migration? Since this isn't in master yet, it would be nice to be part of the `create_table` definition.